test/web-platform-tests/to-upstream/html/browsers/browsing-the-web/navigating-across-documents/fragment-anchors.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
```

Comment:
This one passes in browsers!! \o/. Again, interested in if we can enable any of the existing tests though.